ui/src/app/styles/cards.css

```diff
@@ -7,3 +7,117 @@
 .card * {
   margin-top: 0rem;
 }
+
+.card.item-card {
+  width: 200px;
+  height: 223px;
+  margin-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  cursor: pointer;
+  padding: 0.75rem;
+}
+
+.card.add-card {
+  width: 200px;
+  margin-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  cursor: pointer;
+  padding: 0px;
+  height: 105px;
+  color: #216FB4;
+  box-shadow: 0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12);
+  background-color: #FFFFFF;
+}
+
+.card.add-card.disabled {
+  cursor: not-allowed;
+  color: #AAAFB4;
+}
+
+.card.add-card .create-text {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 22px;
+  position: relative;
+  margin-top: 10%;
+  margin-left: 12%;
+  cursor: pointer;
+}
+
+.card.add-card .plus-circle {
+  margin-top: 5px;
+  width: 21px;
+  height: 21px;
+  display: block;
+}
+
+.card.item-card .footer {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: -0.75rem;
+  margin-right: -0.75rem;
+}
+
+.card.item-card .name {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 22px;
+  color: #216FB4;
+}
+
+.card.item-card .item-type {
+  background-color: #F8C954;
+  height: 42px;
+  width: 42px;
+  box-shadow: 0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12);
+  border-radius: 5px 0 4px 0;
+  box-sizing: border-box;
+  border: 2px solid #DCAE3B;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card.item-card .item-type.cohort {
+  background-color: #F8C954;
+  border: 2px solid #DCAE3B;
+}
+
+.card.item-card .item-type.notebook {
+  background-color: #8BC990;
+  border: 2px solid #6EAF73;
+}
+
+.item-type * {
+  font-size: 22px;
+  font-weight: bold;
+}
+
+.item-type.cohort * {
+  color: #FCE6B0;
+}
+
+.item-type.notebook * {
+  color: #CAECCD;
+}
+
+.card.item-card .last-modified {
+  color: #4A4A4A;
+  font-size: 11px;
+  display: inline-block;
+  line-height: 14px;
```

Comment:
More of question than comment, why are we not using rem for line-height?

Reply:
This is copy and pasted over from invision. We need to have a larger question with Lou about whether it is possible for us to get rem values from invision, otherwise I think we may need to move over to px.